sklearn/metrics/tests/test_classification.py

```diff
@@ -324,6 +324,13 @@ def test_cohen_kappa():
     y2 = np.array([0] * 52 + [1] * 32 + [2] * 16)
     assert_almost_equal(cohen_kappa_score(y1, y2), .8013, decimal=4)
 
+    # Weighting example: none, linear, quadratic.
+    y1 = np.array([0] * 46 + [1] * 44 + [2] * 10)
```

Comment:
@agramfort   There sames no need to make dtype float.
And the above kappa examples in test_classification.py dont use float dtype